state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -161,6 +161,8 @@ pub mod pallet {
 
 		/// The action to be taken when the DepositChannel is deposited to.
 		pub action: ChannelAction<T::AccountId>,
+		/// The boost fee, if set to 0 it means no required
```

Comment:
I was a little confused by this too - does mean 'can't be boosted' or 'no reward for boosting'?

Reply:
The way it will be implemented is that there's no reward. We'll also exclude any 0 fee channels in the subscription. Theoretically could be boosted, but doubt anyone will do so.